resources/Arduino/boardlist.json

```diff
@@ -3,6 +3,7 @@
         {
             "name": "IoT DevKit",
             "id": "devkit",
+            "fqbn": "AZ3166:stm32f4:MXCHIP_AZ3166",
```

Comment:
the fqbn is Arduino special, how about other board?

Reply:
`fqbn` is Arduino-specific. Other board don't have this attribute.
I was considering creating a new class like `ArduinoBoard`, which is inherited from the basic `board` class, but this class will has just slightly diffenrences with the basic `board` class - It has a `fqbn` property. Will that be too heavy?